code/modules/spells/spell_types/wizard.dm

```diff
@@ -370,7 +370,7 @@
 				M.electrocute_act(80, src, illusion = 1)
 		qdel(src)
 
-/obj/item/spellpacket/lightningbolt/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback)

```

Comment:
Would you mind updating arguments here to base ones, it's missing a few from what i can tell.

Reply:
Just add the arguments missing here from atom/movable define.

Reply:
Ahh okay. It looks like only `force = INFINITY` is missing.

Reply:
Couldn't figure out what you meant since I was looking at my local outdate branch heh